pqarrow/arrowutils/sort.go

```diff
@@ -75,8 +74,7 @@ func SortRecord(r arrow.Record, columns []SortingColumn) (*array.Int32, error) {
 
 // ReorderRecord uses indices to create a new record that is sorted according to
 // the indices array.
-func ReorderRecord(ctx context.Context, mem memory.Allocator, r arrow.Record, indices *array.Int32) (arrow.Record, error) {
-	ctx = compute.WithAllocator(ctx, mem)
+func ReorderRecord(ctx context.Context, r arrow.Record, indices *array.Int32) (arrow.Record, error) {
```

Comment:
Can you add a comment about how to pass in an allocator now that it's not explicit?

Reply:
done f8f4d58